### ISSUE TITLE
Little version Zenject

### ISSUE THREAD

**Issue by Abedron** [ssannandeji/Zenject-2019#403]
Hi,
Zenject is very usefful but, is very robust. Can a smaller version be created (remove from librarie)? We need only DI, without Signal, Poller, Tickables and many other uselessness, which occupies memory and CPU only.

**Comment by svermeulen** [comment]
Yeah I actually don't think this is a bad idea.  Most of that stuff is independent of zenject core.  Splitting it out into separate packages should be doable so definitely something to consider for future versions

**Comment by svermeulen** [comment]
By the way this can be done somewhat already by just unchecking the OptionalExtras directory when importing.  But support for the interfaces like ITickables are always added in the Context classes, so not easy to make those optional
